config/manager/manager.yaml

```diff
@@ -65,8 +65,12 @@ spec:
               value: quay.io/jetstack/cert-manager-controller:v1.15.2
             - name: RELATED_IMAGE_CERT_MANAGER_ACMESOLVER
               value: quay.io/jetstack/cert-manager-acmesolver:v1.15.2
+            - name: RELATED_IMAGE_CERT_MANAGER_ISTIOCSR
+              value: quay.io/jetstack/cert-manager-istio-csr:v0.12.0
             - name: OPERAND_IMAGE_VERSION
```

Comment:
since you've added healthz probe endpoints (via controller-runtime), would want to get them added in the deployment spec.